notesScript.js

```diff
@@ -16,6 +16,16 @@ function addNote(notes)
         noteDiv.classList.add('note');
```

Comment:
function `addNote` doesnt really need a parameter, it can just access const `notes` directly